### PR TITLE
CLOUD-1433 Setup nginx exporter for *-web

### PR DESCRIPTION
**Pull request by svenwltr**
> https://jira.rebuy.de/browse/CLOUD-1433

Example: https://grafana.production.rebuy.cloud/dashboard/db/nginx?orgId=1&from=1517925903652&to=1517929347418

### CODE REVIEW

**Comment by svenwltr** [comment]
@rebuy-de/prp-nginx-exporter Please review.

File: assets/supervisord.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    %(ENV_NGINX_ACCESS_LOGS)s
+
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+
+[program:status]
+command=/usr/local/bin/nginx_exporter
+    -nginx.scrape_uri=%(ENV_NGINX_STATUS_URI)s
```
Comment (bethge) [inline assets/supervisord.conf:20]: What does the trailing s do?

**Review by bethge — APPROVED** [review]

File: assets/supervisord.conf (hunk 1 above)
Comment (svenwltr) [reply assets/supervisord.conf:20]: This is Python-string-formatting-foo.

**Comment by svenwltr** [comment]
This doesn't work as expected. I added a note and would merge it anyway. See https://github.com/rebuy-de/web/pull/1221 for reasons.

**Comment by svenwltr** [comment]
Also, bump.

**Review by octopusx — APPROVED** [review]

**Review by stephanlindauer — APPROVED** [review]

**Pull request merged**